runtime/src/snapshot_bank_utils.rs

```diff
@@ -2497,28 +2474,19 @@ mod tests {
             ..snapshot_config
         };
 
-        let genesis_config = GenesisConfig::default();
-        let mut bank = Arc::new(Bank::new_for_tests(&genesis_config));
-
-        // take some snapshots, and archive them
-        for _ in 0..snapshot_config
+        let num_snapshots_to_create = snapshot_config
             .maximum_full_snapshot_archives_to_retain
-            .get()
-        {
-            let slot = bank.slot() + 1;
-            bank = Arc::new(Bank::new_from_parent(bank, &Pubkey::default(), slot));
-            bank.fill_bank_with_ticks_for_tests();
-            bank_to_full_snapshot_archive_with(
-                &snapshot_config.bank_snapshots_dir,
-                &bank,
-                snapshot_config.snapshot_version,
-                &snapshot_config.full_snapshot_archives_dir,
-                &snapshot_config.incremental_snapshot_archives_dir,
-                snapshot_config.archive_format,
-                false,
-            )
-            .unwrap();
-        }
+            .get();
+
+        // take some snapshots, and archive them. Do not flush or hard link storages so that
+        // get highest loadable can be tested when the snapshot has not been marked loadable
```

Comment:
nit: The comment indicates the snapshots will be archived, but I don't think that's accurate anymore.